.coderabbit.yaml

```
@@ -73,5 +74,4 @@ chat:
   auto_reply: true
 
 # Early access features
-early_access:
-  enabled: false
\ No newline at end of file
+early_access: false
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)